src/contexts/index.ts

```diff
@@ -25,5 +25,6 @@ export const RefreshControlComponentContext = React.createContext<
 >(undefined);
 
 export const DocContext = React.createContext<{
-  getDoc: () => TypesLegacy.Document;
+  getDoc: () => TypesLegacy.Document | undefined;
+  setDoc: (doc: TypesLegacy.Document) => void;
```

Comment:
since `hv-screen` consumer does not explicitly set this key, we probably should make it optional
```suggestion
  setDoc?: (doc: TypesLegacy.Document) => void;
```